src/library/scala/util/control/ControlThrowable.scala

```diff
@@ -36,4 +30,6 @@ package util.control
  *
  *  @author Miles Sabin
  */
-trait ControlThrowable extends Throwable with NoStackTrace
+abstract class ControlThrowable(message: String) extends RuntimeException(message, null, false, false) {
```

Comment:
I'm curious as to why it extends `RuntimeException` and not `Throwable`

Reply:
I wondered why anyone would extend Throwable, which is distinguished as Error, RuntimeException, and everything else which is "checked" for Java. Even Scala code cares about Error.

Reply:
I guess the rationale might be that it's something that can be thrown, but doesn't indicate that anything's wrong - therefore extending `Exception` or `Error` doesn't make a lot of sense? idk

Reply:
Oh, I see what you mean. That's a good point.

It should extend `They say never use exceptions for flow control but I'm doing it anyway`, that in backticks, of course.

Reply:
Could you please add an overload that lets the subclass choose the value of `writableStackTrace`. I need [`UndefinedBehaviorError`](https://github.com/scala-js/scala-js/blob/0.6.x/library/src/main/scala/scala/scalajs/runtime/UndefinedBehaviorError.scala) to be a subclass of `ControlThrowable` in 0.6.x, but to still have stack traces.

Reply:
@sjrd Thanks for the use case. BTW, a while ago, I dipped my toe in the scala.js water (or rip current) for work, and I still intend to get back to it, even if I drown trying.

Reply:
FWIW, in Scala.js 1.x `UndefinedBehaviorError` is not a `ControlThrowable`. It's become a `VirtualMachineError` instead.